docs/en/Configuring-Alluxio-with-GCS.md

```diff
@@ -19,6 +19,11 @@ Then, if you haven't already done so, create your configuration file from the te
 
 {% include Common-Commands/copy-alluxio-env.md %}
```

Comment:
this is no longer accurate, the bootstrap-conf should be the recommended mechanism ... the alternative should be that users copy the `alluxio-env.sh.template` and set the contents manually